cli/src/commands/hash.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+/* eslint-disable no-console */
+import { withDevice } from "@ledgerhq/live-common/lib/hw/deviceAccess";
+import { deviceOpt } from "../scan";
+import { from } from "rxjs";
+import axios from "axios";
+import Btc from "@ledgerhq/hw-app-btc";
+import { findCurrencyExplorer } from "@ledgerhq/live-common/lib/api/Ledger";
+import { findCryptoCurrencyById } from "@ledgerhq/live-common/lib/data/cryptocurrencies";
+
+const command = async (transport, currencyId, hash) => {
+  const btc = new Btc(transport);
+  const currency = findCryptoCurrencyById(currencyId);
```

Comment:
(minor) you can also add an `invariant(currency.family==="bitcoin", "currency of bitcoin family only")`